action.yml

```diff
@@ -100,6 +104,17 @@ runs:
         ref: ${{ fromJSON(fromJSON(github.event.inputs.client_payload)).cmRepoRef }}
         path: "gitstream/cm/"
 
+    - name: Docker Layer Caching
+      if: ${{ inputs.pull_docker_from_cache == 'true' }}
+      uses: satackey/action-docker-layer-caching@v0.0.11
+      continue-on-error: true
+      with:
+        concurrency: 12
+
+    - name: Pull gitstream Docker
```

Comment:
It will cache all dockers? How does it knows which docker to cache?

Reply:
It will cache all docker-layers inside **this** workflow, so actually, since we use only `rules-engine` - it will cache only him.